core/modules/simpletest/tests/image.test

```diff
@@ -323,14 +322,14 @@ class ImageToolkitGdTestCase extends BackdropWebTestCase {
       $operations += array(
         'rotate_5' => array(
           'function' => 'rotate',
-          'arguments' => array(5, 0xFF00FF), // Fuchsia background.
+          'arguments' => array(5, '0xFF00FF'), // Fuchsia background.
```

Comment:
This change shouldn't be necessary. If it is, it's an API change. The code may need to adjust to the old argument list if needed.